mlir/test/Dialect/Vector/test-scalable-bounds.mlir

```diff
@@ -0,0 +1,140 @@
+// RUN: mlir-opt %s -test-affine-reify-value-bounds -cse -verify-diagnostics \
+// RUN:   -verify-diagnostics -split-input-file | FileCheck %s
+
+#fixedDim0Map = affine_map<(d0)[s0] -> (-d0 + 32400, s0)>
```

Comment:
[nit] The names of these maps are quite long - why not `#map_dim_i = affine_map<(d0)[s0] -> (-d0 + 32400, s0)>`? That would be easier to find relevant maps. Same comment for other maps.